src/ui/components/Library/LibraryDropdown.tsx

```diff
@@ -35,7 +35,7 @@ export function Dropdown({
 }: {
   children: React.ReactNode;
   menuItemsClassName?: string;
-  widthClass?: "w-56" | "w-64" | "w-80";
+  widthClass?: "" | "w-56" | "w-64" | "w-80";
```

Comment:
Not sure why we wanted bigger-than-necessary menus but the mockups for this context menu seem to want it to tight-fit-content so...